Mathlib/Tactic/Find.lean

```diff
@@ -1,33 +1,95 @@
 /-
 Copyright (c) 2021 Sebastian Ullrich. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Sebastian Ullrich
+Authors: Sebastian Ullrich, Joachim Breitner
 -/
 import Lean
+import Std.Lean.Delaborator
 import Mathlib.Tactic.Cache
 
 /-!
 # The `#find` command and tactic.
+-/
 
-The `#find` command finds definitions & lemmas using pattern matching on the type. For instance:
-```lean
-#find _ + _ = _ + _
-#find ?n + _ = _ + ?n
-#find (_ : Nat) + _ = _ + _
-#find Nat → Nat
-```
-Inside tactic proofs, there is a `#find` tactic with the same syntax,
-or the `find` tactic which looks for lemmas which are `apply`able against the current goal.
+open Lean Meta Elab
 
+/-!
+## Utilities, to be moved somewhere else before merging
 -/
```

Comment:
This is still TODO, I think! :-)

Reply:
Right. I guess I should boldly make a suggestion where they can go :-)